src/components/ModalAlertNotSupported.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import ModalAlert from './ModalAlert.js';
+
+
+/**
+ * Component that shows a message that is not supported
+ *
+ * @memberof Components
+ */
+class ModalAlertNotSupported extends React.Component {
+  render() {
+    const renderBody = () => {
+      return (
+        <div>
+          <p>Unfortunately this feature is not yet supported in this wallet version when using a hardware wallet.</p>
```

Comment:
Suggestion: "Unfortunately this feature is currently not supported when using a hardware wallet."

Reply:
Done